tests/links_tests/connection_tests/test_conv_2d_activ.py

```diff
@@ -93,7 +94,7 @@ def test_forward_gpu(self):
     def check_backward(self, x_data, y_grad):
         x = chainer.Variable(x_data)
         y = self.l(x)
-        y.grad = y_grad
+        y.grad = y_grad if self.dilate == 1 else y_grad[:, :, 1:-1, 1:-1]
```

Comment:
How about

```
if self.dilate == 1:
     y.grad = y_grad
elif self.dilate == 2:
     y.grad = y_grad[:, :, 1:-1, 1:-1]
```

This is better because ` y_grad[:, :, 1:-1, 1:-1]` would not work when `dilate > 2`.

Reply:
OK, I fixed it, but there's no test case here for dilate > 2.